docs/misc/prune-example.txt

```diff
@@ -101,6 +118,8 @@ the one from 2015-12-17.
 
 We did not include weekly, hourly, minutely or secondly rules to keep this
 example simple. They all work in basically the same way.
+Borg 1.2 onwards now "explains" which rule was used to keep a specific archive.
+Use `--list` or `-v`.
```

Comment:
Did you try that for each of these when used alone?

Reply:
> Did you try that for each of these when used alone?
No, I took your word for it.

>BTW, we could mention...(1.1.x is EOL, so won't get docs updates any more).
Not sure I follow. Is there something you'd like me to modify?

Reply:
Well, I used "or" because I was not sure myself what's sufficient to get this output.

About the "1.2" I meant you could as well leave that away as anybody who reads the 1.2 or master docs likely has that functionality already.